### PR TITLE
Clean up GridField

### PR DESCRIPTION
**Pull request by assertchris**

### CODE REVIEW

**Comment by assertchris** [comment]
@tractorcow ping

**Comment by tractorcow** [comment]
refactor

File: forms/gridfield/GridField.php [hunk 1]
```diff
@@ -894,21 +1107,22 @@ public function __construct(GridField $gridField, $name, $title, $actionName, $a
...
 
 	/**
-	 * The callback for nameEncode
+	 * @param array $match
 	 *
-	 * @param string $val
+	 * @return string
 	 */
-	public function _nameEncode($match) {
+	public function _nameEncode(array $match) {
```
Comment (tractorcow) [inline forms/gridfield/GridField.php]: Is this an explicit API change of a public method? Probably should remove that for now.
Why are you deleting the method description?

**Comment by assertchris** [comment]
Fixed

**Comment by dhensby** [comment]
sorry; do you mind, i need some time alone with this...

**Comment by dhensby** [comment]
Though, @assertchris, your comments don't have the stepping of laravel:

**Comment by dhensby** [comment]
I only write comments
where each line is
exactly 3 chars
shorter than
the last

**Pull request merged**